Moderation-Policy.md

```diff
@@ -1,154 +1,288 @@
-
-Any Collaborator found to be violating the privacy of the nodejs/moderation repository by repeatedly sharing or discussing the details of nodejs/moderation issues in any public forum or social media service risks being removed from the Node.js GitHub organization through standard TSC motion and vote.
-
-### TSC Delegation of Authority to a "Moderation Working Group"
-
-For any Moderation issue that does not directly involve a TSC member, the TSC may choose to delegate some or all of it's Moderation-related responsibilities to a "Moderation Working Group". All members of such a Working Group must be Collaborators and the TSC will have responsibility for selecting the membership of that Moderation Working Group.
-
-### Modifications to This Policy
-
-Modifications to this policy are made through normal [TSC motion and vote](https://GitHub.com/nodejs/TSC/blob/master/TSC-Charter.md#section-8-voting). Any Collaborator may submit a PR proposing changes to this policy. Those PRs must be labeled using the `tsc-agenda` label. Including a mention to `@nodejs/tsc` can be used to call the issue to TSC's attention.
+* Moderation of Posts authored by non-Collaborators may result in those
+  non-Collaborators being Banned temporarily or permanently from further
+  participation in the Node.js GitHub organization.
+* In the case where a GitHub Account appears to have been created with no
+  intention to collaborate in good faith, swift actions may be taken without
+  following the above procedures including: removing Posts, Banning permanently,
+  and reporting accounts to GitHub.
+
+Note that Moderating non-Collaborator posts can often lead to retaliation or
+escalation of inappropriate behavior by the individual whose post is being
+Moderated. This is true primarily of individuals whose intent is to harass,
+disrupt or annoy individual members of the community. In such cases, it is best
+to handle the Moderation as quickly and as quietly as possible without drawing
+any further undue attention to the Post in question.
+
+### Temporary Interaction Limits
+
+The Moderation Team may, at their discretion, choose to enable GitHub's
+[Temporary Interaction Limits][] on any GitHub repository in the Node.js GitHub
+Organization.
+
+## Privacy of the nodejs/moderation Repository
+
+The nodejs/moderation Repository is used to discuss the potentially sensitive
+details of any specific moderation issue. The repository is private but
+accessible to all Collaborators. The details of any issue discussed within the
+nodejs/moderation repository are expected to remain confidential and are not to
+be discussed in any public forum or social media service.
+
+Any Collaborator found to be violating the privacy of the nodejs/moderation
+repository by repeatedly sharing or discussing the details of nodejs/moderation
+issues in any public forum or social media service risks being permanently
+Removed from the Node.js GitHub organization.
+
+## Moderation Team
```

Comment:
I'm not 100% sure we need a moderation team.

Perhaps this should be reworked to be based on the third party arbitration mentioned in #318